.changeset/funny-pianos-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-react': patch
+---
+
+Fix filter bug that leaves filters selected when they are not available for the selected kind.
```

Comment:
I hope I did not miss anything in this sentence, the important part was to highlight the specific picker.

```suggestion
Fixed bug in `EntityTagPicker` that filtered on unavailable tags for the selected kind.
```

Reply:
That makes sense! Is it possible to also mention the other pickers that were updated in this PR https://github.com/backstage/backstage/pull/15058 since it did not have a changeset when it was merged?